spec/all_pages_spec.rb

```diff
@@ -51,6 +51,17 @@ def get_locale(path)
         end
       end
 
+      it 'has correctly configured system-status meta tag' do
```

Comment:
I don't know that it's blocking especially since there's already a lot of precedent for testing expected tags (e.g. analytics), but it feels a little wasteful to run this spec example for every single page in the site, if we could alternatively test it once as part of the main "layout" that's assumed to be present for every page.